<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1209</width>
    <height>715</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <italic>true</italic>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{

          background-color:rgb(52,51,64);

}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: blue;
font: 75 35pt &quot;MS Shell Dlg 2&quot;;

</string>
    </property>
    <property name="text">
     <string>MUSIKY</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_filename">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>471</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
border:2px solid rgb(255,255,255);
color:rgb(255,255,255);
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="text">
     <string>Votre musique en cours s'affiche ici........</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>501</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:horizontal{
height:10px;
width:460px;
background:orange;
border-radius:5px;
}

QSlider::handle:horizontal{
background:rgb(255,255,255);
height:20px;
width:20px;
margin:-7px  -7px;
border-radius:10px;



}</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Play">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>400</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
   border:none;
   border-radius:40px;
background-color:rgb(255,255,255);
color:rgb(52,51,64);
min-width:80px;
max-width:80px;
min-height:80px;
max-height:80px;
padding:0px;
}

QPushButton:Hover{
  background-color: yellow 
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3_Pause">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>400</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
   border:none;
   border-radius:40px;
background-color:rgb(255,255,255);
color:rgb(52,51,64);
min-width:80px;
max-width:80px;
min-height:80px;
max-height:80px;
padding:0px;
}

QPushButton:Hover{
  background-color: purple; 
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4_Stop">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>400</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
   border:none;
   border-radius:40px;
background-color:rgb(255,255,255);
color:rgb(52,51,64);
min-width:80px;
max-width:80px;
min-height:80px;
max-height:80px;
padding:0px;
}

QPushButton:Hover{
  background-color: gray;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>520</y>
      <width>501</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:horizontal{
height:10px;
width:460px;
background:green;
border-radius:5px;
}

QSlider::handle:horizontal{
background:rgb(255,255,255);
height:20px;
width:20px;
margin:-7px  -7px;
border-radius:10px;



}</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_volume">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>510</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
   border:none;
   border-radius:40px;
background-color:rgb(255,255,255);
color:rgb(52,51,64);
min-width:40px;
max-width:40px;
min-height:40px;
max-height:40px;
padding:0px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>520</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Volume:</string>
    </property>
   </widget>
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>40</y>
      <width>331</width>
      <height>641</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3_next">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>400</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
   border:none;
   border-radius:40px;
background-color:rgb(255,255,255);
color:rgb(52,51,64);
min-width:80px;
max-width:80px;
min-height:80px;
max-height:80px;
padding:0px;
}

QPushButton:Hover{
  background-color: yellow 
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_previous">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>400</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
   border:none;
   border-radius:40px;
background-color:rgb(255,255,255);
color:rgb(52,51,64);
min-width:80px;
max-width:80px;
min-height:80px;
max-height:80px;
padding:0px;
}

QPushButton:Hover{
  background-color: yellow 
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>230</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:Hover{
  background-color: blue; 
}</string>
    </property>
    <property name="text">
     <string>Ajouter une chanson à la bibliothèque</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_start">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_end">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>150</y>
      <width>47</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>150</y>
      <width>311</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>90</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Afficher les chansons</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>160</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Afficher les listes de lecture</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1209</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menugtt">
    <property name="title">
     <string>Ouvrir</string>
    </property>
    <addaction name="horizontalSlider_Audio_File_Duration"/>
    <addaction name="actionListe_de_lecture"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionR_initialiser_la_biblioth_que"/>
    <addaction name="actionRaffraichir_la_binlioth_que"/>
    <addaction name="actionQuitter"/>
   </widget>
   <addaction name="menugtt"/>
   <addaction name="menuOptions"/>
  </widget>
  <action name="horizontalSlider_Audio_File_Duration">
   <property name="text">
    <string>Fichier musique</string>
   </property>
  </action>
  <action name="actionR_initialiser_la_biblioth_que">
   <property name="text">
    <string>Réinitialiser la bibliothèque</string>
   </property>
  </action>
  <action name="actionRaffraichir_la_binlioth_que">
   <property name="text">
    <string>Raffraichir la bibliothèque</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionListe_de_lecture">
   <property name="text">
    <string>Liste de lecture</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
